moped-database/migrations/1696693713827_asset_feature_layer_id/up.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE moped_components
+    ADD COLUMN asset_feature_layer_id integer
+    REFERENCES feature_layers (id)
+    ON UPDATE CASCADE ON DELETE SET NULL;
+
+COMMENT ON COLUMN moped_components.asset_feature_layer_id IS
+    'Foreign key which indicates if the component supports storing data from a reference asset layer, and in which layer that data should be stored';
+
+-- make default layer for signals and phbs be intersections
+-- and set asset layer to signals 
+UPDATE
+    moped_components
+SET
+    feature_layer_id = 5,
+    asset_feature_layer_id = 3
```

Comment:
Signal and phb components will reference the  `feature_intersections` layer just like all other point components, and now there is a new reference column (`asset_feature_layer_id`) specifically for which layer to use for asset data.

This should make easier to scale out to additional asset types like flashing beacons and school zones (both of which have been requested by AMD).